### PR TITLE
Application/healthcare chatbot

### PR DESCRIPTION
**Pull request by akashAD98**
Healthcare related  LLM 

Biomedllm lama3 b llm
neuml pubbert -embedding model
cohere reranker 
lancedb with langchain

### CODE REVIEW

File: applications/Healthcare_chatbot/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+## Overview
+This project introduces a healthcare-related Retrieval-Augmented Generation (RAG) chatbot, designed to deliver quick  responses to medical inquiries. Utilizing  OpenBioLLM-Llama3 / openai llm and the NeuML's PubMedBERT for embedding,
+this chatbot is adept at processing and responding to medical data queries.
+
+![image](https://github.com/akashAD98/vectordb-recipes/assets/62583018/6027ccb2-0917-45e4-97f4-6739aa420082)
```
Comment (PrashantDixit0) [inline applications/Healthcare_chatbot/README.md:5]: Change it to relative path ../assets/chatbot_medical.png

File: applications/Healthcare_chatbot/main.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+docs = loader.load()
+logging.info("Document loader done.")
+
+# Set up the text processing and model chain
+#llm = ChatOpenAI(model="gpt-4", temperature=0, openai_api_key=OPENAI_API_KEY)
+
+# download weights from https://huggingface.co/PrunaAI/OpenBioLLM-Llama3-8B-GGUF-smashed/tree/main
+llm = LlamaCpp(
+    model_path="/content/OpenBioLLM-Llama3-8B.Q4_K_S.gguf",  # path of gguf weight
```
Comment (PrashantDixit0) [inline applications/Healthcare_chatbot/main.py:54]: How this .gguf file has to be downloaded?

File: applications/Healthcare_chatbot/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+
+## Key Features
+### Language Model: 
+
+To utilize OpenBioLLM-Llama3 .download model in the local system & pass the path of it 
+link for downloading ggmf version model https://huggingface.co/PrunaAI/OpenBioLLM-Llama3-8B-GGUF-smashed
```
Comment (PrashantDixit0) [inline applications/Healthcare_chatbot/README.md:12]: Typo GGMF to gguf

File: applications/Healthcare_chatbot/README.md (hunk 1 above)
Comment (akashAD98) [reply applications/Healthcare_chatbot/README.md]: done

File: applications/Healthcare_chatbot/main.py (hunk 2 above)
Comment (akashAD98) [reply applications/Healthcare_chatbot/main.py:54]: added it in readme

**Pull request merged**